app1/models.py

```diff
@@ -1,3 +1,18 @@
+# myapp/models.py
 from django.db import models
```

Comment:
**Static Code Issue**: Import block is un-sorted or un-formatted <br> **Fix**: Sort the imports properly <br> **Code Suggestion**: 
 ```
 - from django.db import models
 + from django.db import models
 ```